Explorer/Assets/Scripts/CrdtEcsBridge/RestrictedActions/RestrictedActionsAPIImplementation.cs

```diff
@@ -77,14 +77,6 @@ public bool TryChangeRealm(string message, string realm)
             return true;
         }
 
-        public void TryTriggerEmote(string predefinedEmote)
```

Comment:
Why this got deleted?

Reply:
Apparently when I merged main it got in the crossfire

Reply:
I restored it